.circleci/config.yml

```diff
@@ -1,145 +1,376 @@
+    parallelism: "<< parameters.parallelism >>"
+    steps:
+      - ci-utils/ci_checkout
+      - bundle_install:
+          cache-dependencies: "<< parameters.cache-dependencies >>"
+          cache-salt: "<< parameters.ruby-version >>"
+      - install_gem_version:
+          gem-version: "<< parameters.gem-version >>"
+      - steps: "<< parameters.pre-actions >>"
+      - steps: "<< parameters.actions >>"
+      - steps: "<< parameters.post-actions >>"
+      - ci-utils/quietly_store_artifacts
 
 workflows:
-  trunk:
-    jobs:
-      - build:
-          <<: *master_only
-          matrix:
-            parameters:
-              executor: [ruby_3_0, ruby_3_1, ruby_3_2]
-      - gem/build:
-          <<: *master_only
-          executor: ruby_3_2
-          name: gem-build
-          requires:
-            - build
-
-  pull-requests:
-    jobs:
-      - build:
-          <<: *pr_only
-          matrix:
-            parameters:
-              executor: [ruby_3_0, ruby_3_1, ruby_3_2]
-      - gem/build:
-          <<: *pr_only
-          executor: ruby_3_2
-          name: gem-build
-          vendor-cache: false
-          requires:
-            - build
-      - pre-release-approval:
-          <<: *pr_only
-          type: approval
-          requires:
-            - gem-build
-      - gem/publish:
-          <<: *pr_only
-          name: gem-publish
-          to_rubygems: false
-          to_nexus: true
-          pre_release: true
-          requires:
-            - pre-release-approval
-          context: artifact_publishing
-
-  final-release:
+  main:
     jobs:
-      - build:
-          <<: *version_tags_only
+      - ruby_mvt_job:
+          executor: ruby-with-services
+          context: nexus_readonly
+          post-actions:
+            - store_test_results:
+                path: tmp/test-results
+          actions:
+            - run_rspec_tests:
+                test_pattern: "{$(ls -d spec/**/ | sed -E 's/(spec\\/|factories|support|\\/|,$)//g'
+                  | sort | uniq | tr '\\n' ',' | sed 's/,\\{2,\\}/,/g')}"
+                test_files: $(circleci tests glob "spec/$TEST_PATTERN/**/*_spec.rb" |
+                  circleci tests split --split-by=timings)
+          parallelism: 1
           matrix:
             parameters:
-              executor: [ruby_3_0, ruby_3_1, ruby_3_2]
-      - gem/build:
-          <<: *version_tags_only
-          executor: ruby_3_2
-          name: gem-build
-          vendor-cache: false
-          requires:
-            - build
-      - gem/publish:
-          <<: *version_tags_only
-          name: gem-publish
-          to_rubygems: true
-          pre_release: false
-          requires:
-            - gem-build
-          context: artifact_publishing
+              ruby-version:
+                - '3.1'
+                - '3.2'
+                - '3.3'
+              gem-version:
+                - rails-6.1.7
+                - rails-7.0.8
+                - rails-7.1.3
```

Comment:
Rails 7.1 tests currently fail: https://app.circleci.com/pipelines/github/doximity/super_spreader?branch=jjh%2FIA-2150%2Fruby-rails-mvt

Reply:
It has something to to with `Minitest`, `assert`, etc.  It's not completely obvious to me why that is.

Reply:
What do you think about removing Rails 7.1 from this matrix for now and making a new issue to follow up on the error?

Reply:
Good idea. Filed https://github.com/doximity/super_spreader/issues/60